keyvalue/client/pom.xml

```diff
@@ -9,19 +9,16 @@
   ~ See the NOTICE file distributed with this work for information regarding copyright ownership.
   -->
 
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
   <modelVersion>4.0.0</modelVersion>
   <parent>
     <groupId>org.alluxio</groupId>
     <artifactId>alluxio-keyvalue</artifactId>
     <version>1.1.0-SNAPSHOT</version>
   </parent>
-  <groupId>org.alluxio</groupId>
   <artifactId>alluxio-keyvalue-client</artifactId>
-  <packaging>jar</packaging>
-  <name>Alluxio Key Value - Client Distribution</name>
-  <description>Distribution of client for Alluxio Key Value Store</description>
+  <name>Alluxio Key Value - Client Implementation</name>
```

Comment:
Now that we don't have to distinguish between Client Implementation and Client Distribution, can we just call it Client?